### PR TITLE
move the package pyautogen to autogen

### PR DESCRIPTION
**Pull request by Hk669**
## Why are these changes needed?

moving the package from `pyautogen` to `autogen` for all the existing notebooks in the cookbook. 

## Checks

- [ ] I've included any doc changes needed for https://autogen-ai.github.io/autogen/. See https://autogen-ai.github.io/autogen/docs/Contribute#documentation to build and test documentation locally.
- [ ] I've added tests (if relevant) corresponding to the changes introduced in this PR.
- [ ] I've made sure all auto checks have passed.

### CODE REVIEW

File: cookbook/oai_chatgpt_gpt4.ipynb [hunk 1]
```diff
@@ -1,2195 +1,2195 @@
...
-     "iopub.status.busy": "2023-02-13T23:40:54.633929Z",
-     "iopub.status.idle": "2023-02-13T23:40:56.105700Z",
-     "shell.execute_reply": "2023-02-13T23:40:56.105085Z"
+  "cells": [
+    {
+      "attachments": {},
+      "cell_type": "markdown",
+      "metadata": {},
+      "source": [
+        "<a href=\"https://colab.research.google.com/github/microsoft/autogen/blob/main/notebook/oai_chatgpt_gpt4.ipynb\" target=\"_parent\"><img src=\"https://colab.research.google.com/assets/colab-badge.svg\" alt=\"Open In Colab\"/></a>"
```
Comment (sonichi) [inline cookbook/oai_chatgpt_gpt4.ipynb]:
```suggestion
        "<a href=\"https://colab.research.google.com/github/autogen-ai/autogen/blob/main/notebook/oai_chatgpt_gpt4.ipynb\" target=\"_parent\"><img src=\"https://colab.research.google.com/assets/colab-badge.svg\" alt=\"Open In Colab\"/></a>"
```

File: cookbook/oai_completion.ipynb [hunk 2]
```diff
@@ -1,1202 +1,1202 @@
...
-     "name": "stderr",
-     "output_type": "stream",
-     "text": [
-      "Found cached dataset openai_humaneval (/home/vscode/.cache/huggingface/datasets/openai_humaneval/openai_humaneval/1.0.0/2955cebd73602e828fa8c0a424c594e5fab4ec863b316ca98f3d8fdb6a626e75)\n"
-     ]
+      "attachments": {},
+      "cell_type": "markdown",
+      "metadata": {},
+      "source": [
+        "<a href=\"https://colab.research.google.com/github/microsoft/autogen/blob/main/notebook/oai_completion.ipynb\" target=\"_parent\"><img src=\"https://colab.research.google.com/assets/colab-badge.svg\" alt=\"Open In Colab\"/></a>"
```
Comment (sonichi) [inline cookbook/oai_completion.ipynb]:
```suggestion
        "<a href=\"https://colab.research.google.com/github/autogen-ai/autogen/blob/main/notebook/oai_completion.ipynb\" target=\"_parent\"><img src=\"https://colab.research.google.com/assets/colab-badge.svg\" alt=\"Open In Colab\"/></a>"
```

**Review by sonichi — COMMENTED** [review]
The change for notebooks can be made in the autogen repo for now.

**Review by marklysze — APPROVED** [review]
Looks good :)

**Pull request merged**